data/org.freedesktop.portal.Audio.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2021 Endles OS Foundation, LLC
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.Audio:
+      @short_description: Audio portal
+
+      The audio portal enables applications to access audio devices, such as
+      microphones and speakers.
+  -->
+  <interface name="org.freedesktop.portal.Audio">
+    <!--
+        AccessAudio:
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the @handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Request documentation for
+              more information about the @handle.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+
+        Following the #org.freedesktop.portal.Request::Response signal, if
+        granted, #org.freedesktop.org.Audio.OpenPipeWireRemote can be used to
+        open a PipeWire remote.
+    -->
+    <method name="AccessAudio">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
```

Comment:
Is there no way for the app to express that it is only interested in input, or only in output ?

Reply:
I thought a bit about that, and decided to not add until requested by any reviewed :slightly_smiling_face: 

I'll add that

Reply:
Added